blocklylib-core/src/main/res/values/attrs.xml

```diff
@@ -31,6 +32,7 @@
 
     <declare-styleable name="WorkspaceFragment">
         <attr name="scrollable" format="boolean"/> <!-- Default true -->
+        <attr name="scalable" format="boolean"/> <!-- Default true -->
```

Comment:
Combine these into one zoomBehavior enum.  See comments in virtualWorkspaceView for details.